Simplenote/Classes/SPNoteTableViewCell.swift

```diff
@@ -0,0 +1,248 @@
+            accessoryImageView.tintColor = newValue
+        }
+    }
+
+    /// Number of Maximum Preview lines to be rendered
+    ///
+    var numberOfPreviewLines: Int {
+        get {
+            return previewTextView.textContainer.maximumNumberOfLines
+        }
+        set {
+            previewTextView.textContainer.maximumNumberOfLines = newValue
+        }
+    }
+
+    /// Note's Text
+    ///
+    var previewText: NSAttributedString? {
+        get {
+            return previewTextView.attributedText
+        }
+        set {
+            previewTextView.attributedText = newValue
+        }
+    }
+
+    /// Note's Text Alpha Value
+    ///
+    var previewAlpha: CGFloat {
+        get {
+            return previewTextView.alpha
+        }
+        set {
+            previewTextView.alpha = newValue
+        }
+    }
+
+    /// Returns the Preview's Fragment Padding
+    ///
+    var previewLineFragmentPadding: CGFloat {
+        return previewTextView.textContainer.lineFragmentPadding
+    }
+
+    /// Designated Initializer
+    ///
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    // MARK: - Overridden Methods
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        setupTextView()
+        setupImageView()
+        setupSubviews()
+        setupStackView()
+        refreshStyle()
+    }
+
+    /// Returns the Preview TextView's frame in the specified view's coordinates.
+    ///
+    /// TODO: Nuke this. And forgive me for pushing it, in the first place.
+    ///
+    @objc
+    func previewFrame(in view: UIView) -> CGRect {
+        return view.convert(previewTextView.frame, from: previewTextView)
+    }
+
+    /// Highlights the partial matches with the specified color.
+    ///
+    func highlightSubstrings(matching keywords: String, color: UIColor) {
+        previewTextView.textStorage.apply(color, toSubstringMatchingKeywords: keywords)
+    }
+}
+
+
+// MARK: - Private Methods: Initialization
+//
+private extension SPNoteTableViewCell {
+
+    /// Setup: TextView
+    ///
+    func setupTextView() {
+        previewTextView.isScrollEnabled = false
+        previewTextView.isUserInteractionEnabled = false
+        previewTextView.isEditable = false
+        previewTextView.isAccessibilityElement = false
+        previewTextView.backgroundColor = .clear
+        previewTextView.textContainerInset = .zero
+
+        let container = previewTextView.textContainer
+        container.maximumNumberOfLines = Options.shared.numberOfPreviewLines
+        container.lineFragmentPadding = 0
+        container.lineBreakMode = .byWordWrapping
+    }
+
+    /// Setup: ImageView
+    ///
+    func setupImageView() {
+        accessoryImageView.contentMode = .center
+    }
+
+    /// Setup: StackView
+    ///
+    func setupStackView() {
+        accessoryStackView.addArrangedSubview(accessoryImageView)
+    }
+
+    /// Autolayout Init
+    ///
+    func setupSubviews() {
```

Comment:
Maybe I'm missing something but it seems like most (all?) of the work being done in `setupTextView`, `setupImageView`, `setupStackView`, and `setupSubviews` could be done in the xib vs in code. I'm not sure defining `previewTextView` and `accessoryImageView` as lazy props vs @IBOutlets is offering much of an optimization. Seems like its just off-loading work into `awakFromNib` that otherwise would have already been done when the views were decoded. Thoughts?

Reply:
I'm afraid... this isn't an optimization, **but** the only way I could set things up. 

Reasoning is:`SPTextView` uses a custom **textContainer**, and [I couldn't manage to implement an `initWithCoder` method, capable of replacing the textContainer](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/Classes/SPTextView.m#L31).

As per **setupStackView** and **setupImageView**, I'll revisit those in an upcoming PR, which is aaalmost done. **Definitelly** can and will simplify that.

**Thank you Eric!!!**